utils/debug-utils.stanza

```diff
@@ -490,17 +491,11 @@ public defn add-testpoint (ref:JITXObject, cmp:Instantiable, persist?:True|False
         if not empty?(pins) :
           net (ref, pins[0])
       else if persist? and not property(self.debug) :
-        if top? :
-          inst tp : top-smd-testpoint-cmp()
-          schematic-group(tp) = debug-persist-sch
-          layout-group(tp) = debug-persist-layout
-          net (ref, tp.p)
-        else :
-          inst tp : btm-smd-testpoint-cmp()
-          schematic-group(tp) = debug-persist-sch
-          layout-group(tp) = debug-persist-layout
-          net (ref, tp.p)
-
+        inst tp : smd-testpoint-cmp()
```

Comment:
Thanks for removing the `top-smd-test-point-cmp` and `bottom-smd-test-point-cmp` definitions. Whether a component is on the top or bottom of a board should not be baked into the component definitions themselves. 

Here is the command that you can use to re-enable support for test points on the bottom.

```
inst tp : smd-testpoint-cmp()
place(tp) on Bottom
```